lib/airbrake/shoryuken/error_handler.rb

```diff
@@ -0,0 +1,34 @@
+module Airbrake
+  module Shoryuken
+    ##
+    # Provides integration with Shoryuken.
+    class ErrorHandler
+      def call(_worker, _queue, _sqs_msg, body)
+        yield
+      rescue => e
+        notify_airbrake(e, notice_context(body))
+        raise e
+      end
+
+      private
+
+      def notice_context(body)
+        body.is_a?(Array) ? { batch: body } : { body: body }
```

Comment:
>This way we don't execute it when notify_airbrake returns early (and we so far we try to use this style in other integrations).

@kyrylo I'm moving it to the `notify_airbrake` method, but it won't help with that ^, because it needs to be passed in the `build_notice` call. So even for the earliest it can return, `notice_context` will be called.